checkita-core/src/main/scala/ru/raiffeisen/checkita/config/validation/PostValidation.scala

```diff
@@ -216,6 +216,21 @@ object PostValidation {
       refPathFilter = (s: String) => s.contains(".kafka.")
     )
 
+  /**
+   * Validation to check if DQ job configuration contains missing references
+   * from table sources to pivotal connections
+   */
+  val validateGreenplumSourceRefs: ConfigObject => Vector[ConfigReaderFailure] = root =>
+    validateCrossReferences(
+      getObjOrEmpty(root, "jobConfig.sources"),
+      getObjOrEmpty(root, "jobConfig.connections"),
+      "connection",
+      "id",
+      "Greenplum source refers to undefined pivotal connection '%s'",
+      checkPathPrefix = "jobConfig.sources",
+      refPathPrefix = "jobConfig.connections"
+    )
```

Comment:
add checkPath filter to validate only greenplum sources and also add refPath filter to search for pvitoal greenplum connections only.